packages/notification-services-controller/src/NotificationServicesController/ui/constants.ts

```diff
@@ -0,0 +1,23 @@
+import { NOTIFICATION_CHAINS_ID } from '../constants/notification-schema';
+
+export const NOTIFICATION_NETWORK_CURRENCY_NAME = {
+  [NOTIFICATION_CHAINS_ID.ETHEREUM]: 'Ethereum',
+  [NOTIFICATION_CHAINS_ID.ARBITRUM]: 'Arbitrum',
+  [NOTIFICATION_CHAINS_ID.AVALANCHE]: 'Avalanche',
+  [NOTIFICATION_CHAINS_ID.BSC]: 'Binance',
+  [NOTIFICATION_CHAINS_ID.LINEA]: 'Linea',
+  [NOTIFICATION_CHAINS_ID.OPTIMISM]: 'Optimism',
+  [NOTIFICATION_CHAINS_ID.POLYGON]: 'Polygon',
+} as const;
+
+export const NOTIFICATION_NETWORK_CURRENCY_SYMBOL = {
+  [NOTIFICATION_CHAINS_ID.ETHEREUM]: 'ETH',
+  [NOTIFICATION_CHAINS_ID.ARBITRUM]: 'ETH',
+  [NOTIFICATION_CHAINS_ID.AVALANCHE]: 'AVAX',
+  [NOTIFICATION_CHAINS_ID.BSC]: 'BNB',
+  [NOTIFICATION_CHAINS_ID.LINEA]: 'ETH',
+  [NOTIFICATION_CHAINS_ID.OPTIMISM]: 'ETH',
+  [NOTIFICATION_CHAINS_ID.POLYGON]: 'MATIC',
+};
+
+export { NOTIFICATION_CHAINS_ID } from '../constants/notification-schema';
```

Comment:
Mobile does not have consistent token name/exports to extension. To keep things in sync, we will use these shared constants in this package.